### ISSUE TITLE
Some Sort of Neutral->Angry 0-Damage Solution

### ISSUE THREAD

**Issue by Dynamiczbee** [Rebirth-of-the-Night/onslaught#37]
Hey, I've been working around with TFC and Onslaught for a while and I've realized that I've come upon a bit of a hole with the ability for the mod to add angry AI to neutral entities in TFC. My thought is if there is some way to cause an attack upon the entities spawning with the source being defined as the player such that the neutral enemies become hostile with targeting against players. I've been able to emulate this by summoning snowballs with the ownerName NBT tag equal to the players name but it's a really janky solution. Thought I'd ask and see if it was at all possible.

TLDR; Method to cause "damage" to entity upon spawning in invasion with source (see ownerName NBT) being listed as the invasion's target player.
